src/controllers/AnnotationModeController.js

```diff
@@ -133,7 +133,7 @@ class AnnotationModeController extends EventEmitter {
     destroy(): void {
         Object.keys(this.annotations).forEach((pageNum) => {
             const pageThreads = this.annotations[pageNum].all() || [];
-            pageThreads.forEach(this.unregisterThread);
+            pageThreads.forEach(this.unregisterThread.bind(this));
```

Comment:
Ideally, it seems like this should be bound elsewhere, maybe in the constructor?